resolvers.js

```diff
@@ -319,6 +368,41 @@ const resolvers = {
       return newCategory;
     },
 
+    createDatabaseCategoryForCurrentNote: async (
+      parent,
+      { databaseId, categoryName, noteId },
+      context
+    ) => {
```

Comment:
add comments above this function. it is not obvious what this does thru its name